gen/installer/azure.py

```diff
@@ -150,8 +149,7 @@ def gen_templates(user_args, arm_template):
         results.templates[arm_template + '.json'],
         variant_cloudconfig['master'],
         variant_cloudconfig['slave'],
-        variant_cloudconfig['slave_public']
-        )
+        variant_cloudconfig['slave_public'])
```

Comment:
This is one of two allowed styles currently:

```python
     # Render the arm
     arm = render_arm(
          results.templates[arm_template + '.json'],
          variant_cloudconfig['master'],
          variant_cloudconfig['slave'],
          variant_cloudconfig['slave_public'])
```
and

```python
     # Render the arm
     arm = render_arm(
          results.templates[arm_template + '.json'],
          variant_cloudconfig['master'],
          variant_cloudconfig['slave'],
          variant_cloudconfig['slave_public']
    )
```

Are both currently allowed. We can toggle between the second and:
```python
     # Render the arm
     arm = render_arm(
          results.templates[arm_template + '.json'],
          variant_cloudconfig['master'],
          variant_cloudconfig['slave'],
          variant_cloudconfig['slave_public']
        )
```

but only one of those two can be allowed at a time.